documents/Libraries/stdlib/stdlib_defs.mtlx

```diff
@@ -262,12 +245,8 @@
     <parameter name="layery" type="string" value=""/>
     <parameter name="layerz" type="string" value=""/>
     <parameter name="default" type="float" value="0.0"/>
-    <input name="position" type="vector3">
-      <geomprop name="position" type="vector3"/>
-    </input>
-    <input name="normal" type="vector3">
-      <geomprop name="normal" type="vector3"/>
-    </input>
+    <input name="position" type="vector3" defaultgeomprop="Pobject"/>
```

Comment:
Should these not be world space position and normal?

Reply:
The current behavior was to use object space here. Because object space is default space for "position" and "normal" nodes, and no modifier was used in this case.

I _think_ object space is correct, as lease for position, since you don't want the projection to change if you translate or scale an object. But the user is free to override this by connecting a position/normal node with other space set.

Reply:
I guess both can be correct. Let's leave it as is.